R/make_om_part_B.R

```diff
@@ -0,0 +1,34 @@
+#' Make Outcome Measures Part B
+#'
+#' @description Award Status at Four Years after Entry
+#'
+#' @param df A dataframe of student statuses
+#' @param output A string (\code{"part"}, \code{"full"}, or \code{"both"})
+#' @param format A string (\code{"uploadable"}, \code{"readable"}, or \code{"both"})
+#'
+#' @return A text file ready for IPEDS upload
+#' @export
+#'
+#'
+make_om_part_B <- function(df, output = "part", format = "both") {
+
+    partB <- prep_om_awards(df, award = "AwardLevel4") %>%
```

Comment:
Why do you only have AwardLevel14?

Reply:
part b does 4 year grad (part c does 5 year grad rates and part d does 6 year grad rates)